app/src/main/res/layout/activity_main.xml

```diff
@@ -1,23 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
+              xmlns:app="http://schemas.android.com/apk/res-auto"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:orientation="vertical">
 
-    <TextView
-        android:layout_width="match_parent"
-        android:layout_height="400px"
-        android:text="You are logged in!" />
+    <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="vertical">
 
-    <Button
-        android:id="@+id/logoutButton"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:text="Log Out!" />
+        <TextView
+                android:layout_width="match_parent"
+                android:layout_height="400px"
+                android:text="You are logged in!"/>
 
-    <Button
-        android:id="@+id/mapButton"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:text="Open Map" />
+        <Button
+                android:id="@+id/logoutButton"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="Log Out!"/>
+    </LinearLayout>
+
+    <fragment
+            android:name="io.imhungry.fragments.BottomNavigationFragment"
+            android:id="@+id/bottomNavFragment"
+            android:layout_width="match_parent"
+            android:layout_height="50dp"/>
```

Comment:
Put the fragment in a separate XML file, and use an `include` element instead. This way we don't have to update all instances whenever there are changes.